src/utils/utils.tsx

```diff
@@ -58,7 +58,8 @@ export const daysInMonth = (date: string) => {
 }
 
 const intervalsList = ['year', 'month', 'week', 'day', 'hour']
-export const getClosestInterval = (range: any) => {
+
+export const getClosestInterval = (range: Range): Interval => {
```

Comment:
use `Terms` interface instead of `Range`, it includes these fields along with `Interval` type so no need to cast

Reply:
Nice, I was not aware of Terms interface. But casting is still needed here:
```
 if (duration[intervalPlural] || 0 > 1) {
            if (intervalsList.indexOf(interval) > intervalsList.indexOf(selectedInterval)) {
                selectedInterval = interval as Interval
            }
            return true
        }
```

since interval is string and we need to assign it as an Interval type. Otherwise linter throws error

Reply:
just add `Interval[]` type to `intervalsList`

Reply:
I did, thanks